widgetastic_manageiq/__init__.py

```diff
@@ -1831,9 +1833,8 @@ def selected(self):
             return names[0]
 
     def select(self, name):
-        button = self._get_button(name)
         if self.selected != name:
-            button.click()
+            self.browser.element('.//input[@type="radio"]', parent=self._get_button(name)).click()
```

Comment:
**REQUIRED**: `self.browser.click('.//input[@type="radio"]', parent=self._get_button(name))` so you don't bypass the WT facilities.